slrealizer/data/dataloader.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function
+
+import os
+
+class Dataloader:
```

Comment:
This looks like a nice clean solution - certainly it streamlines the tests! I guess the notebooks might need updating too, but that's a separate issue...

Reply:
Yay! I was waiting to hear how you liked it before making more changes :smiley: 

For future reference, Phil has issued #50 for continuously integrating demo notebooks.